tests/base.py

```diff
@@ -251,8 +252,7 @@ def setUp(self):
         self.app = TestAppJSONAPI(api_django_app)
 
 
-@override_settings(DEBUG_PROPAGATE_EXCEPTIONS=True)
```

Comment:
This should be specific to the test cases or the yellow and grey exception error page with the trace turns into plain/text and everyone is :disappointed: